src/main/java/com/scottlogic/practice/mealscheduler/Services/MealServiceConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package com.scottlogic.practice.mealscheduler.Services;
+
+import com.scottlogic.practice.mealscheduler.Repositories.MealRepo;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MealServiceConfiguration {
+    @Bean
+    MealService createMealService(MealRepo mealRepo) {
+        return new MealService(mealRepo);
+    }
+}
```

Comment:
Is this not exactly what adding `@Service` does? I know you said you thought it was better for the future but for now I don't understand what benefit this adds except another file to look after?